client/src/hooks/useEmployeeSchedule.ts

```diff
@@ -31,9 +35,17 @@ type createEmployeeScheduleMutationType = UseMutationResult<
   unknown
 >
 
+type editEmployeeScheduleMutationType = UseMutationResult<
```

Comment:
Reminder: All types must start with Capital Letter
```suggestion
type EditEmployeeScheduleMutationType = UseMutationResult<
```